src/main/java/ua/ithillet/lesson6/phonebook/Phonebook.java

```diff
@@ -0,0 +1,25 @@
+package ua.ithillet.lesson6.phonebook;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Phonebook {
+    private List<RecordOnPhone> recordOnPhoneList = new ArrayList<>();
+
+    public List<RecordOnPhone> getRecordOnPhoneList() {
```

Comment:
```suggestion
    protected List<RecordOnPhone> getRecordOnPhoneList() {
```
Оскільки ти відкриваєш цей метод лише для тестів, то краще хоча б прикрити його через protected.

Reply:
Але тоді я не зможу використовувати його в тестах, бо це інший пакет. Чи треба використовувати наслідування?